guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -35,43 +35,52 @@ The process for configuring Fastly includes:
 * Advanced configurations including VCL snippets
 * Enable and configure the Fastly image optimization feature (Fastly IO)
 
-## Multiple Fastly accounts and assigned domains {#domain}
 
-Before launching {{site.data.var.ece}}, you may already have a Fastly account
-or trial with your apex and subdomains assigned to it. Be advised, you must
-remove any apex domain and subdomains you plan to use with {{site.data.var.ece}}
-from this existing Fastly account.
+## Get Fastly credentials {#cloud-fastly-creds}
 
-Fastly only allows one apex domain and all subdomains assigned to a single Fastly
-service and account. For example, if you have the apex domain of mystore.com with
-subdomains of shoes.mystore.com and socks.mystore.com managed by an existing
-Fastly account, you need to remove them from that account before going live with
-Fastly and {{site.data.var.ece}}.
+{{ site.data.var.ece }} projects do not require a dedicated Fastly account or account owner. Instead, each project environment has its own set of Fastly credentials (API key and service ID) that you use to configure and manage Fastly services from the Magento Admin UI. You also use these credentials to submit requests to the Fastly API service. 
 
-For details, review your Fastly accounts and [documentation](https://docs.fastly.com/)
-to remove the domains. This may include removing and updating CNAME records and
-more.
+To get these credentials, Magento needs to add your project environments to the Fastly service account for {{ site.data.var.ece }} and add the Fastly account credentials to your project environment configurations.
```

Comment:
When it says "add your project environments", does it mean you are adding a link?

Reply:
Yes-- link is the right concept. Thanks.